README.md

```diff
@@ -3,40 +3,34 @@
 
-This repository contains modular implementations of the [storage and retrieval market subsystems](https://filecoin-project.github.io/specs/#systems__filecoin_markets) of Filecoin. They are guided by the [v1.0 and 1.1 Filecoin specification updates](https://filecoin-project.github.io/specs/#intro__changelog). 
+This repository contains modular implementations of the [storage and retrieval market subsystems](https://filecoin-project.github.io/specs/#systems__filecoin_markets) of Filecoin. 
+They are guided by the [v1.0 and 1.1 Filecoin specification updates](https://filecoin-project.github.io/specs/#intro__changelog). 
 
 Separating implementations into a blockchain component and one or more mining and market components presents an opportunity to encourage implementation diversity while reusing non-security-critical components.
 
 ## Components
 
-* **[filestore](./filestore)**: a submodule branch that is a side effect of using the ffi to generate commP
-* **[pieceio](./pieceio)**: utilities that take IPLD graphs and turn them into pieces
-* **[piecestore](/.piecestore)**:  
-* **[storage market](./storagemarket)**: for finding, negotiating, and consummating deals to store data between clients and providers (storage miners)
-* **[retrieval market](./retrievalmarket)**: for finding, negotiating, and consummating deals to retrieve data between clients and providers (retrieval miners)
+* **[filestore](./filestore)**: a submodule branch that is a side effect of using the ffi to
+ generate commP.
+* **[pieceio](./pieceio)**: utilities that take IPLD graphs and turn them into pieces.
+* **[piecestore](./piecestore)**:  a database for storing deal-related PieceInfo and CIDInfo. 
+* **[storagemarket](./storagemarket)**: for finding, negotiating, and consummating deals to
+ store data between clients and providers (storage miners).
+* **[retrievalmarket](./retrievalmarket)**: for finding, negotiating, and consummating deals to
+ retrieve data between clients and providers (retrieval miners).
 
 Related components in other repos:
 * **[data transfer](https://github.com/filecoin-project/go-data-transfer)**: for exchanging piece data between clients and miners, used by storage & retrieval market modules.
 
 ### Background reading
-* The [Markets in Filecoin](https://filecoin-project.github.io/specs/#systems__filecoin_markets) section of the Filecoin Specification contains the canonical spec
-* The [Storage Market Module design doc](https://docs.google.com/document/d/1FfMUpW8vanR9FrXsybxBBbba7DzeyuCIN2uAXgE7J8U) is a more specific overview of these component implementations
-
-## Usage
-
-**Requires go 1.13**
-
-Install the go-fil-markets module(s) that you want to use.  Available modules are:
-* `filestore`
-* `pieceio`
-* `piecestore`
-* `retrievalmarket`
-* `storagemarket`
+* The [Markets in Filecoin](https://filecoin-project.github.io/specs/#systems__filecoin_markets) 
+section of the Filecoin Specification contains the canonical spec.
 
```

Comment:
```suggestion
## Install
```

Reply:
I used Installation since it fits the other titles (e.g. Operation) and also because it's what I used in other READMEs.